pymongo/pool.py

```diff
@@ -211,13 +211,14 @@ def _set_keepalive_times(sock: socket.socket) -> None:
         "version": platform.mac_ver()[0],
     }
 elif sys.platform == "win32":
+    _ver = sys.getwindowsversion()
     _METADATA["os"] = {
-        "type": platform.system(),
-        # "Windows XP", "Windows 7", "Windows 10", etc.
-        "name": " ".join((platform.system(), platform.release())),
-        "architecture": platform.machine(),
-        # Windows patch level (e.g. 5.1.2600-SP3)
-        "version": "-".join(platform.win32_ver()[1:3]),
+        "type": "Windows",
+        "name": "Windows",
+        # Avoid using platform calls, see PYTHON-4455.
+        "architecture": os.environ.get("PROCESSOR_ARCHITECTURE") or platform.machine(),
+        # Windows patch level (e.g. 10-0-17763).
```

Comment:
is `10-0-17763` accurate? I would expect something like `10.0.17763-XXX`

Reply:
I need to update the comment, I just pushed a change to make it format the same as the old way.